hivemind/server/module_registrator.py

```diff
@@ -0,0 +1,24 @@
+import typing as tp
+import functools
+import torch
+import torch.nn as nn
+
+from hivemind.server.layers import name_to_block, name_to_input
```

Comment:
Why not 
1) add this directly to hivemind.server.layers
2) register existing expert layers with this interface for uniformity?